ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -562,14 +567,35 @@ impl Module {
         self.loop_start_vars.push(op);
     }
 
+    pub(crate) fn set_parent_entry_vars(&mut self, entry_vars: &[VarLocation]) {
```

Comment:
Add doc strings for all new functions please.

Reply:
[9a39a8b](https://github.com/ykjit/yk/pull/1422/commits/9a39a8bb1c45f9128f50360f7dfa1391c4da6516)